bench-tests/bt-0000-0001/README.md

```diff
@@ -8,16 +8,31 @@ random read/write.
 
 The tests that are run using smallfile are, create, read, ls -l (listing)
 
-## Pre-requsites
+## Pre-requisites
   - The environment variable CLIENTS must be set and contain a space separated
-    list of IPs or hostnames
+    list of IPs or hostnames [1]
   - The environment variable SERVERS must be set and contain a space separated
-    list of IPs or hostnames
+    list of IPs or hostnames [1]
   - The iozone config file must be stored in /root/clients.ioz.  It also must
-    have 4 threads per client
-  - The smallfile application must be installed in /root/smallfile
-  - Passwordless SSH should be setup across the CLIENTS and SERVERS
+    have 4 threads per client [2]
+  - The smallfile application must be installed in /root/smallfile. If you have git installed this step will be performed during execution.
+  - Passwordless SSH should be setup across the CLIENTS and SERVERS [3]
   - Provided sync-drop-caches.sh should be copied to /root/sync-drop-caches.sh
+  
+[1] To set the env. variable temporarily for the current shell and all the processes started from the current shell:
+
+``` export CLIENTS=your-client-ip-01\ your-client-ip-02 #Spcae seperated list of IPs ```
+
+and
+
+``` export SERVERS=your-server-ip-01\ your-server-ip-02  #Spcae seperated list of IPs ```
+  
+[2] The configuration file contains a series of records that look like this: ``` hostname   directory   iozone-pathname ```
+
+Where hostname is a host name or IP address of a test driver machine that iozone can use, directory is the pathname of a directory to use within that host, and iozone-pathname is the full pathname of the iozone executable to use on that host. Be sure that every target host can resolve the hostname of host where the iozone command was run. All target hosts must permit password-less ssh access from the host running the command.
+
```

Comment:
Add:
Each record is a thread in iozone terminology, so as this test by default uses 4 threads, there should be 4 records per client mentioned in the CLIENTS environment variable.